aws_s3_rules/aws_s3_unauthenticated_access.py

```diff
@@ -1,10 +1,13 @@
 # A list of buckets where authenticated access is expected
-AUTH_BUCKETS = {"example-bucket"}
+from panther_aws_helpers import aws_rule_context
 
+AUTH_BUCKETS = {"example-bucket"}
```

Comment:
We shouldn't have comments at the very top of the file unless it's for a license

The line
`from panther_aws_helpers import aws_rule_context`
Should be on line 1 and the comment current there should be above `AUTH_BUCKETS`